tools/mountsnoop.py

```diff
@@ -341,6 +374,8 @@ def print_event(mounts, umounts, cpu, data, size):
                 'mnt_ns': event.union.enter.mnt_ns,
                 'comm': event.union.enter.comm,
                 'flags': event.union.enter.flags,
+                'ppid': event.ppid if event.ppid > 0 else get_ppid(event.pid),
+                'pcomm': pid_to_comm(event.ppid),
```

Comment:
You can get pcomm in bpf program. This way, you don't need to worry about short lived processes.

Reply:
Thanks.We will revise it later according to your suggestions.